primitives/src/eip4844.rs

```diff
@@ -0,0 +1,87 @@
+//! Helpers for working with EIP-4844 blob fee.
+
+#[cfg(feature = "c-kzg")]
```

Comment:
instead of all these `c-kzg` conditions, maybe possible to just conditional include the whole file?

Reply:
I copied and pasted so idk if we want keep it as it is or combine

Reply:
If the code works in zkVM I think we may even want to remove the feature because raiko will require it in all guests, so fine with me either way now.